jupyterlab/environment.yaml

```diff
@@ -61,3 +61,20 @@ dependencies:
       - jupyterlab-conda-store >=0.2.7
       - jupyter-tensorboard
       - jupyterlab_nvdashboard
+      # pyvista requirements
+      - cmocean
+      - colorcet
+      - imageio
+      - imageio-ffmpeg
+      - ipygany
+      - ipyvtklink
+      - jupyter-server-proxy
+      - meshio
+      - panel
+      - pythreejs
+      - trame
+      - trame-client>=2.4.2
+      - trame-server>=2.7.1
+      - trame-vtk>=2.4.0
+      - trimesh
+      - vtk
```

Comment:
I will follow up soon with a specific, custom build of VTK to use for this environment. I'm hoping to make one wheel that will  "just work" for the CPU and GPU nodes

Reply:
For the CPU nodes, the best bet is to use the OSMesa build of VTK from conda-forge: `vtk=*=osmesa*`

... still working on the GPU node